packages/server/src/sdk/app/rows/sqlAlias.ts

```diff
@@ -156,12 +157,19 @@ export default class AliasTables {
   }
 
   async queryWithAliasing(
-    json: QueryJson
+    json: QueryJson,
+    queryFn?: (json: QueryJson) => Promise<DatasourcePlusQueryResponse>
   ): Promise<DatasourcePlusQueryResponse> {
     const datasourceId = json.endpoint.datasourceId
-    const datasource = await sdk.datasources.get(datasourceId)
+    const isSqs = datasourceId === SQS_DATASOURCE_INTERNAL
+    let aliasingEnabled: boolean, datasource: Datasource | undefined
+    if (isSqs) {
+      aliasingEnabled = true
```

Comment:
Should this respect the value of the environment variable that disables aliasing?

Reply:
I'm not sure about this one - the env variable is mostly for allowing users to disable it if they encounter a problem, with SQS I don't know if we want there to be two internal flows.

But also it could be quite useful for ourselves to have the ability to turn it on and off in the Cloud if we think it was causing a problem. I'll add it in just incase.

Reply:
Don't mind which direction you go with it but I follow your reasoning. I'd probably err on the side of leaving it until we have a use-case, but I'm not strongly either way.